build-standalone

```diff
@@ -19,19 +19,20 @@ function flatten() {
   # required for running the image (ENV, EXPOSE, WORKDIR, etc) so we
   # manually rebuild them.
   # See here for more details: https://github.com/moby/moby/issues/8334
-  local container=`docker create $image`
+  local container
```

Comment:
Merely curious, but what's the significance of declaring the variable on a separate line?

Reply:
https://github.com/koalaman/shellcheck/wiki/SC2155

Reply:
Very interesting, TIL! Thanks!